apps/browser/qml/pages/components/FavoriteGrid.qml

```diff
@@ -38,6 +43,8 @@ IconGridViewBase {
         }
     }
 
+    onCountChanged: height = !searched ? startHeight : count ? cellHeight + headerItem.height : headerItem.height
```

Comment:
This would be cleaner if you could bind to the `height` property directly.

Reply:
if you use the condition in `height`, then` count` will be processed incorrectly